examples/pdo_poller_example.py

```diff
@@ -0,0 +1,86 @@
+import time
+from typing import Dict, List, Union
+
+from ingeniamotion.enums import SensorType
+from ingeniamotion.motion_controller import MotionController
+
+
+def establish_coe_connection(mc: MotionController) -> None:
+    """Establish an EtherCAT-CoE communication.
+
+    Find all available nodes, and perform a communication.
+
+    Args:
+        mc: The object where there are all functions to establish a communication.
+    """
+    # Modify these parameters to connect a drive
+    interface_index = 3
+    slave_id = 1
+    dictionary_path = "parent_directory/dictionary_file.xdf"
+
+    interface_selected = mc.communication.get_ifname_by_index(interface_index)
+    slave_id_list = mc.communication.scan_servos_ethercat(interface_selected)
+
+    if not slave_id_list:
+        interface_list = mc.communication.get_interface_name_list()
+        print(f"No slave detected on interface: {interface_list[interface_index]}")
+        return
+    else:
+        print(f"Found slaves: {slave_id_list}")
+
+    mc.communication.connect_servo_ethercat(interface_selected, slave_id, dictionary_path)
+    print("Drive is connected.")
+
+
+def set_feedback_sensors(mc: MotionController) -> None:
+    """Set the type of position and velocity feedback sensors.
+
+    Args:
+        mc: the controller to configure.
+    """
+    # Modify the SensorType.
+    mc.configuration.set_position_feedback(SensorType.HALLS)
+    mc.configuration.set_velocity_feedback(SensorType.HALLS)
```

Comment:
Why do you set HALLS sensor position and velocity feedbacks? This is the feedback with the worst resolution, it is an unusual case.